tests/services/base/conftest.py

```diff
@@ -0,0 +1,76 @@
+from inspect import getmembers, isclass
+from types import ModuleType
+
+import pytest
+from airflow.models.connection import Connection
+
+from orca.services import base, nextflowtower, sevenbridges
+
+SERVICES = {
+    nextflowtower: "tower://:foo@api.tower.nf/?workspace=bar/baz",
+    sevenbridges: "sbg://:foo@api.sbgenomics.com/v2/?project=bar/baz",
+}
+
+
+def get_service_name(service_module: ModuleType) -> str:
+    return service_module.__name__.split(".")[-1]
+
+
+@pytest.fixture(params=SERVICES, ids=get_service_name)
+def service(request, patch_os_environ):
```

Comment:
Am I missing in the documentation somewhere that `patch__os_environ` is a provided fixture?  Or is this to support the mocking of the os.environ in the config tests?

Reply:
This is a custom fixture that's implemented in the top-level `conftest.py` file. It's empties the environment to make sure that any connection URIs loaded in the `.env` file are ignored. 

```python
@pytest.fixture
def patch_os_environ(mocker):
    yield mocker.patch("os.environ", {})
```